### ISSUE TITLE
Postcode is not listed in the array of errors

### ISSUE THREAD

**Issue by TomashKhamlai** [magento/graphql-ce#1029]
### Preconditions (*)
No preconditions

### Steps to reproduce (*)

1. Perform mutation as a Customer
```graphql
mutation {
    createCustomerAddress(
      input: {
        country_id: AF
        #country_code: "AF"
        city: "1"
        firstname: "2"
        lastname: "3"
      }
    ) {
      country_id
      #country_code
      city
      firstname
      lastname
    }
  }
```
### Expected result (*)
```graphql
{
  "errors": [
    {
      "message": "Required parameters are missing: street, telephone, postcode",
      "extensions": {
        "category": "graphql-input"
      },
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "createCustomerAddress"
      ]
    }
  ],
  "data": {
    "createCustomerAddress": null
  }
}
```

### Actual result (*)
```graphql
{
  "errors": [
    {
      "message": "Required parameters are missing: street, telephone",
      "extensions": {
        "category": "graphql-input"
      },
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "createCustomerAddress"
      ]
    }
  ],
  "data": {
    "createCustomerAddress": null
  }
}
```

**Comment by osrecio** [comment]
I checked this issue but:

The `postcode` in the table `eav_attribute` the column `is_required` is set to `0`. 

The implementation on the file: `\Magento\CustomerGraphQl\Model\Customer\Address\CreateCustomerAddress::validateData`

```
foreach ($attributes as $attributeName => $attributeInfo) {
            if ($attributeInfo->getIsRequired()
                && (!isset($addressData[$attributeName]) || empty($addressData[$attributeName]))
            ) {
                $errorInput[] = $attributeName;
            }
        }
```
if the `postcode` field is not required, so I think the behaviour is ok.

We have 2 options:
1) Add `is_required` in table `eav_attribute`. This can be a problem... for BC and the other implementations.
2) Add to validate data something like:
```
foreach ($attributes as $attributeName => $attributeInfo) {
            if (($attrtibuteName == 'postcode' && && (!isset($addressData[$attributeName]) || empty($addressData[$attributeName])) || $attributeInfo->getIsRequired()
                && (!isset($addressData[$attributeName]) || empty($addressData[$attributeName]))
            ) {
                $errorInput[] = $attributeName;
            }
        } 
```

**Comment by osrecio** [comment]
I saw the validation after save. Creating a PR

**Comment by magento-engcom-team** [comment]
Hi @TomashKhamlai. Thank you for your report. 
The issue has been fixed in magento/graphql-ce#1031 by @osrecio in 2.3-develop branch
Related commit(s):
 - [676cd1b048aaef37a2716d8c658a81cf2d89afc7](https://github.com/magento/graphql-ce/commit/676cd1b048aaef37a2716d8c658a81cf2d89afc7)
 - [127835e3d9cb4c51bab70291e57a37a12b505d4c](https://github.com/magento/graphql-ce/commit/127835e3d9cb4c51bab70291e57a37a12b505d4c)
 - [891e5a4e91ce5ef261cb22c457210322bf4ddcd9](https://github.com/magento/graphql-ce/commit/891e5a4e91ce5ef261cb22c457210322bf4ddcd9)

The fix will be available with the upcoming 2.3.5 release.
